x-pack/plugins/apm/public/components/shared/environment_filter/index.tsx

```diff
@@ -37,6 +38,9 @@ export function ApmEnvironmentFilter() {
       status={status}
       environment={environment}
       availableEnvironments={environments}
+      serviceName={serviceName}
+      rangeFrom={rangeFrom as string}
+      rangeTo={rangeTo as string}
```

Comment:
What if `rangeFrom` or `rangeTo` is undefined?

Reply:
If they are not defined we will get an error in the page like we get today.
Does that sound reasonable?

Reply:
I think it would be better to return null then if the page is going to break anyway. To be in the safety side:

```js
if(!rangeFrom || !rangeTo){
  return null
}
```
WDYT?